sandbox-web/src/main/java/com/exadel/sandbox/team5/DiscountRestController.java

```diff
@@ -14,6 +16,7 @@
 public class DiscountRestController {
```

Comment:
Let's use the plural for `discount` as well `/discount` -> `/discounts` and tell about that frontend team.

Reply:
Done.